src/view/pod/creation_page.rs

```diff
@@ -21,8 +21,13 @@ use crate::view;
 
 const ACTION_CREATE: &str = "pod-creation-page.create";
 const ACTION_ADD_LABEL: &str = "pod.add-label";
+const ACTION_ADD_HOST: &str = "pod.add-host";
```

Comment:
I started to use the widget name as the first part of the action name (forgot to rename those here...).
This makes it easier to understand in which widget the action is defined (as actions are searched in the root direction of the widget tree and there are some places where widgets use actions of parent widgets).

The same applies to other actions that were added in this PR.

At some point, I will rename the remaining actions in a separate PR.

```suggestion
const ACTION_ADD_HOST: &str = "pod-creation-page.add-host";
```